WordPress/Classes/Extensions/Post+BloggingPrompts.swift

```diff
@@ -14,13 +14,13 @@ extension Post {
             tags = Strings.promptTag
         }
 
-        if FeatureFlag.bloggingPromptsEnhancements.enabled {
```

Comment:
There is a value in having a feature flag instead of `AppConfiguration.isJetpack`: easier to find places where it's conditionally added. It may be worth keeping these flags.

Reply:
Fair point - how about I add a `bloggingPromptsEnhancementsEnabled` property to `AppConfiguration`?

Reply:
Sounds good!

Reply:
Fixed! https://github.com/wordpress-mobile/WordPress-iOS/pull/21499/commits/49fa0204b7f64d0ff210bfcf80c2cc02e8f74755

I've removed the branch here because I'm not differentating between `FeatureFlag.bloggingPrompts` and `FeatureFlag.bloggingPromptsEnhancements` - if `AppConfiguration.bloggingPromptsEnabled = true`, then the enhanced version will be shown